.github/workflows/cd.yaml

```diff
@@ -0,0 +1,56 @@
+# Note: additional setup is required, see https://www.jenkins.io/redirect/continuous-delivery-of-plugins
+#
+# Please find additional hints for individual trigger use case
+# configuration options inline this script below.
+#
+---
+name: cd
```

Comment:
Nitpick: Should we name it as "Continuous Delivery" or "Deploy" which is more descriptive (can update filename accordingly).